site/content/docs/user/nvidia-dranet.md

```diff
@@ -0,0 +1,460 @@
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-2 \
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-3 \
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-4 \
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-5 \
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-6 \
+  --additional-node-network network=${RDMA_NETWORK_PREFIX}-net,subnetwork=${RDMA_NETWORK_PREFIX}-sub-7
+```
+
+Apply the following DaemonSet to install the RDMA binaries and the NCCL library on the node. The RDMA binaries are stored in `/home/kubernetes/bin/gib` directory and the NCCL library is stored in `/home/kubernetes/bin/nvidia/lib64` directory on the VM:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/refs/heads/master/gpudirect-rdma/nccl-rdma-installer.yaml
+```
+
+Install DRANet
+```sh
+kubectl apply -f https://raw.githubusercontent.com/google/dranet/refs/heads/main/install.yaml
+```
+
+#### Installing Nvidia DRA Drivers
+
+In order to install the NVIDIA DRA Drivers you will need to clone the [NVIDIA DRA](https://github.com/NVIDIA/k8s-dra-driver-gpu) repo. Ensure you have [helm](https://helm.sh/docs/intro/install/) installed.
+
+```
+helm upgrade -i --create-namespace --namespace nvidia-dra-driver-gpu nvidia-dra-driver-gpu ./k8s-dra-driver-gpu/deployments/helm/nvidia-dra-driver-gpu --set gpuResourcesEnabledOverride=true --set nvidiaDriverRoot=/home/kubernetes/bin/nvidia --wait 
+```
+
+Once this is done, you can run 
+
+```sh
+k get pods -n nvidia-dra-driver-gpu
+NAME                                                READY   STATUS     RESTARTS   AGE
+nvidia-dra-driver-gpu-controller-66696889cd-86m8f   1/1     Running    0          13m
+```
+If you only see the controller like above, you will need to label the nodes with GPUs on them in order to get the kubelet plugin running.
+
+```sh
+kubectl label node -l cloud.google.com/gke-nodepool=<nodepool-name> --overwrite nvidia.com/gpu.present=true
+
+k get pods -n nvidia-dra-driver-gpu
+NAME                                                READY   STATUS     RESTARTS   AGE
+nvidia-dra-driver-gpu-controller-66696889cd-86m8f   1/1     Running    0          12m
+nvidia-dra-driver-gpu-kubelet-plugin-ffzgx          2/2     Running    0          34s
+nvidia-dra-driver-gpu-kubelet-plugin-qsp2d          2/2     Running    0          33s
+```
+
+Once you see all these pods, the NVIDIA DRA plugin is working as expected
+
+#### Creating a GPU workload
+
+We can create a `ResourceClaimTemplate` to specify what GPUs we want. We currently don't have PCI attributes yet in the NVIDIA driver library so we will want to specify the index for the time being.
```

Comment:
we need a short test before this section that provide some explanation about GPU and NIC alignemnt, so people understand better this `PCI attributes` reference